<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWinClass</class>
 <widget class="QMainWindow" name="MainWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWin.qrc">
    <normaloff>:/icons/media/icons/discharger.ico</normaloff>:/icons/media/icons/discharger.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="LoginWgt" name="authPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="loaderPage">
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>15</number>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="topMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
           <property name="bottomMargin">
            <number>15</number>
           </property>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="loaderLblMovie">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>180</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="loaderLblStatus">
             <property name="text">
              <string>loading ...</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="TestConfigWgt" name="confTestPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="testPage">
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="testPageArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>834</width>
             <height>928</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_27">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="testPageLayout">
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QFrame" name="testFrameConfChart">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="label_26">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Configure the chart</string>
                   </property>
                  </widget>
                 </item>
                 <item alignment="Qt::AlignVCenter">
                  <widget class="QPushButton" name="testBtnConfChart">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>44</width>
                     <height>44</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>44</width>
                     <height>44</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="MainWin.qrc">
                     <normaloff>:/icons/media/icons/chart.ico</normaloff>:/icons/media/icons/chart.ico</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="MenuWgt" name="menuPage">
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDevice_characteristic"/>
    <addaction name="actionDevice_information"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogout"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWgtParams">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowIcon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/parameters.ico</normaloff>:/icons/media/icons/parameters.ico</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ParamsWgt" name="paramsWgt">
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWgtCommFlow">
   <property name="windowIcon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/comm_flow.ico</normaloff>:/icons/media/icons/comm_flow.ico</iconset>
   </property>
   <property name="windowTitle">
    <string>Communication flow</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="CommFlowWgt" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_25">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarTest">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionConfNewTest"/>
   <addaction name="separator"/>
   <addaction name="actionShowTestParams"/>
   <addaction name="actionShowCommFlow"/>
   <addaction name="separator"/>
   <addaction name="actionConfChart"/>
   <addaction name="actionResultsOnline"/>
   <addaction name="actionQrCode"/>
  </widget>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/help.ico</normaloff>:/icons/media/icons/help.ico</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/start.ico</normaloff>:/icons/media/icons/start.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start test</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/stop.ico</normaloff>:/icons/media/icons/stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop test</string>
   </property>
  </action>
  <action name="actionConfNewTest">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/new_test.ico</normaloff>:/icons/media/icons/new_test.ico</iconset>
   </property>
   <property name="text">
    <string>Configure new test</string>
   </property>
   <property name="toolTip">
    <string>Configure new test</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/logout.ico</normaloff>:/icons/media/icons/logout.ico</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionShowCommFlow">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/comm_flow.ico</normaloff>:/icons/media/icons/comm_flow.ico</iconset>
   </property>
   <property name="text">
    <string>Communication flow</string>
   </property>
   <property name="toolTip">
    <string>Show communication flow</string>
   </property>
  </action>
  <action name="actionConfChart">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/chart.ico</normaloff>:/icons/media/icons/chart.ico</iconset>
   </property>
   <property name="text">
    <string>Configure chart</string>
   </property>
   <property name="toolTip">
    <string>Configure test chart</string>
   </property>
  </action>
  <action name="actionShowTestParams">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/parameters.ico</normaloff>:/icons/media/icons/parameters.ico</iconset>
   </property>
   <property name="text">
    <string>Test parameters</string>
   </property>
   <property name="toolTip">
    <string>Show test parameters</string>
   </property>
  </action>
  <action name="actionResultsOnline">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/web.ico</normaloff>:/icons/media/icons/web.ico</iconset>
   </property>
   <property name="text">
    <string>Results online</string>
   </property>
   <property name="toolTip">
    <string>Open results online</string>
   </property>
  </action>
  <action name="actionQrCode">
   <property name="icon">
    <iconset resource="MainWin.qrc">
     <normaloff>:/icons/media/icons/qr_code.ico</normaloff>:/icons/media/icons/qr_code.ico</iconset>
   </property>
   <property name="text">
    <string>QR code</string>
   </property>
   <property name="toolTip">
    <string>Show QR code</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionDevice_characteristic">
   <property name="text">
    <string>Device characteristic</string>
   </property>
  </action>
  <action name="actionDevice_information">
   <property name="text">
    <string>Device information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LoginWgt</class>
   <extends>QWidget</extends>
   <header>loginwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuWgt</class>
   <extends>QWidget</extends>
   <header>MenuWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestConfigWgt</class>
   <extends>QWidget</extends>
   <header>testconfigwgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParamsWgt</class>
   <extends>QWidget</extends>
   <header>paramswgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommFlowWgt</class>
   <extends>QWidget</extends>
   <header>commflowwgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWin.qrc"/>
  <include location="C:/Users/domin/.designer/backup/MainWin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWinClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
